industry_benchmarks/input_structures/prepared_structures/mcs_docking_set/renin/protein.pdb

```diff
@@ -0,0 +1,5555 @@
+ATOM   3159  HA  SER A 201     -27.021   8.674  -8.847  1.00  0.00           H  
+ATOM   3160  HB3 SER A 201     -29.434   7.251  -7.680  1.00  0.00           H  
+ATOM   3161  HB2 SER A 201     -29.059   7.417  -9.395  1.00  0.00           H  
+ATOM   3162  HG  SER A 201     -29.525   9.568  -7.577  1.00  0.00           H  
+ATOM   3163  N   SER A 202     -27.693   8.057  -5.651  1.00  0.00           N  
+ATOM   3164  CA  SER A 202     -27.652   8.580  -4.287  1.00  0.00           C  
+ATOM   3165  C   SER A 202     -26.240   8.737  -3.691  1.00  0.00           C  
+ATOM   3166  O   SER A 202     -25.347   7.940  -3.983  1.00  0.00           O  
+ATOM   3167  CB  SER A 202     -28.567   7.773  -3.361  1.00  0.00           C  
+ATOM   3168  OG  SER A 202     -28.658   6.396  -3.704  1.00  0.00           O  
+ATOM   3169  H   SER A 202     -28.043   7.115  -5.753  1.00  0.00           H  
+ATOM   3170  HA  SER A 202     -28.075   9.583  -4.334  1.00  0.00           H  
+ATOM   3171  HB3 SER A 202     -29.564   8.214  -3.365  1.00  0.00           H  
+ATOM   3172  HB2 SER A 202     -28.216   7.867  -2.334  1.00  0.00           H  
+ATOM   3173  HG  SER A 202     -27.793   6.073  -3.966  1.00  0.00           H  
+ATOM   3174  N   THR A 203     -26.051   9.797  -2.871  1.00  0.00           N  
+ATOM   3175  CA  THR A 203     -24.830  10.108  -2.122  1.00  0.00           C  
+ATOM   3176  C   THR A 203     -24.969   9.345  -0.805  1.00  0.00           C  
+ATOM   3177  O   THR A 203     -25.330   9.913   0.227  1.00  0.00           O  
+ATOM   3178  CB  THR A 203     -24.620  11.640  -1.987  1.00  0.00           C  
+ATOM   3179  OG1 THR A 203     -24.607  12.230  -3.293  1.00  0.00           O  
+ATOM   3180  CG2 THR A 203     -23.324  11.999  -1.250  1.00  0.00           C  
+ATOM   3181  H   THR A 203     -26.795  10.465  -2.729  1.00  0.00           H  
+ATOM   3182  HA  THR A 203     -23.981   9.695  -2.667  1.00  0.00           H  
+ATOM   3183  HB  THR A 203     -25.460  12.054  -1.430  1.00  0.00           H  
+ATOM   3184  HG1 THR A 203     -23.891  11.851  -3.808  1.00  0.00           H  
+ATOM   3185 HG21 THR A 203     -23.231  13.083  -1.185  1.00  0.00           H  
+ATOM   3186 HG22 THR A 203     -23.348  11.577  -0.245  1.00  0.00           H  
+ATOM   3187 HG23 THR A 203     -22.471  11.593  -1.794  1.00  0.00           H  
+ATOM   3188  N   LEU A 204     -24.744   8.029  -0.876  1.00  0.00           N  
+ATOM   3189  CA  LEU A 204     -24.873   7.113   0.247  1.00  0.00           C  
+ATOM   3190  C   LEU A 204     -23.949   7.400   1.443  1.00  0.00           C  
+ATOM   3191  O   LEU A 204     -24.324   7.075   2.573  1.00  0.00           O  
+ATOM   3192  CB  LEU A 204     -24.745   5.635  -0.212  1.00  0.00           C  
+ATOM   3193  CG  LEU A 204     -25.967   4.912  -0.863  1.00  0.00           C  
+ATOM   3194  CD1 LEU A 204     -27.302   5.266  -0.197  1.00  0.00           C  
+ATOM   3195  CD2 LEU A 204     -26.044   5.148
```

Comment:
```suggestion
ATOM   5119  OXT ARG A 326      -2.057   6.763 -21.573  1.00  0.00           O1-
```
I think we need to rename the terminal oxygen to OXT to follow PDB conventions, otherwise the run script was failing for me.